src/libs/actions/Task.js

```diff
@@ -151,6 +152,62 @@ function createTaskAndNavigate(currentUserEmail, parentReportID, title, descript
     Navigation.navigate(ROUTES.getReportRoute(optimisticTaskReport.reportID));
 }
 
+function completeTask(taskReportID, parentReportID, taskTitle) {
+    const message = `Completed task: ${taskTitle}`;
```

Comment:
I am planning on keeping it the same because this is just temporary until the reportAction is created in Auth which doesn't have localization yet. 

So once they fetch the report, it would be english still even if we translated it on create

What do you think?

Reply:
Ah okay good point. Maybe someday we'll have backend translation 😄